### ISSUE TITLE
Can't change the start time of the first timeSlot of the day

### ISSUE THREAD

**Issue by sofokli1** [toggl/superday#907]

**Comment by Odrakir** [comment]
I agree, the UX should show that, maybe making that cell a bit dimmer.

**Comment by pe0ny** [comment]
- [ ] To indicate that first time slot's start time and last time slot's end time are not editable, display those with grey color #909293

## Design Preview
### Edit view: first and last slot disabled
![edit - first slot of the day 1x](https://user-images.githubusercontent.com/5064445/35848064-5603c148-0b25-11e8-9a7f-c307b163c0d0.png) ![edit - last slot of the day 1x](https://user-images.githubusercontent.com/5064445/35848066-57fbcebe-0b25-11e8-9715-25bccc00ab03.png)

Zeplin: https://zpl.io/bl1WyP5 & https://zpl.io/2pxYzJj
